pkg/providers/amifamily/bootstrap/bottlerocket.go

```diff
@@ -29,6 +30,8 @@ import (
 
 type Bottlerocket struct {
 	Options
+	AMIVersion       string
+	AMISelectorTerms []v1.AMISelectorTerm
```

Comment:
Any reason we need `AMISelectorTerms` as well as AMI version here?

Reply:
thanks for the catch, remnant of a refactor; I'll remove